src/client/windshaft.js

```diff
@@ -282,14 +282,14 @@ export default class Windshaft {
     }
 
     _buildSelectClause(MRS, dateFields = []) {
-        return MRS.columns.map(name => R.schema.column.getBase(name)).map(
+        const columns = MRS.columns.map(name => R.schema.column.getBase(name)).map(
```

Comment:
Isn't this `MNS`?

Reply:
oops! but I consistently misnamed it 😛

Reply:
But... Minimum Required Schema sounds even better than Minimum Needed Schema!